src/renderer/components/ui/sendMessageSeparate/SendMessageSeparateInitial.js

```diff
@@ -0,0 +1,301 @@
+        {errors[name]}
+      </Typography>
+    ) : (
+      <span />
+    )
+  }
+  return (
+    <Grid
+      container
+      justify='flex-start'
+      direction='column'
+      className={classes.contentContainer}
+    >
+      <Typography variant='h3' className={classes.title}>
+        New message
+      </Typography>
+      <Typography variant='body2' className={classes.fieldName}>Recipient</Typography>
+      <AutocompleteField
+        freeSolo
+        name={'recipient'}
+        inputValue={values.recipient || ''}
+        options={usersArray.map(option => option.nickname)}
+        filterOptions={(options, state) =>
+          options.filter(o =>
+            o.toLowerCase().includes(values.recipient || ''.toLowerCase())
+          )
+        }
+        value={values.recipient}
+        onChange={(e, v) => {
+          setFieldValue('recipient', v)
+          const shouldSetCheckbox = checkField(v)
+          if (shouldSetCheckbox) {
+            setFieldValue('sendAnonymously', true)
+          }
+        }}
+        onInputChange={(e, v) => {
+          setFieldValue('recipient', v)
+          const shouldSetCheckbox = checkField(v)
+          if (shouldSetCheckbox) {
+            setFieldValue('sendAnonymously', true)
+          }
+        }}
+        renderInput={params => (
+          <TextField
+            {...params}
+            className={classes.gutter}
+            variant='outlined'
+            multiline
+            placeholder={`Enter Zbay username or z-address`}
+            margin='normal'
+            fullWidth
+          />
+        )}
+      />
+      <ErrorText name={'recipient'} />
+      <Grid item className={classes.labelDiv}>
+        <Field
+          name={'sendAnonymously'}
+          component={() => (<FormControlLabel
+            classes={{ root: classes.checkboxForm }}
+            control={
+              <Checkbox
+                checked={shouldSendAnonymously || values.sendAnonymously}
+                disabled={shouldSendAnonymously}
+                onChange={(e, v) => setFieldValue('sendAnonymously', !values.sendAnonymously)}
+                color='default'
+                icon={<Icon src={CheckboxUnchecked} />}
+                checkedIcon={<Icon src={CheckboxChecked} />}
+                className={classes.checkbox}
+              />
+            }
+            label={
+              <Typography variant='body2' className={classes.label}>
+                    Send anonymously
+              </Typography>
+            }
+          />)}
+        />
+      </Grid>
+      {(shouldSendAnonymously || (isUserSelected && values.sendAnonymously)) && (
+        <Grid container item>
+          <Grid className={classes.infoBox} item>
+            <Typography className={classes.typo} variant='body2'><span className={classes.bold}>Warning:</span>{` anonymity in Zcash has limits. Lorem ipsum where we explain this
+            anonymity in Zcash has limits. Lorem ipsum where we explain this,
```

Comment:
Lorem ipsum ? Or just placeholder?

Reply:
The text should be:

Warning: anonymity in Zcash has limits. For example, an attacker may be able to discover your IP address or link this message to your other activity on Zbay.